app/main.ipynb

```diff
@@ -56,31 +57,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 94,
    "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "arr = [0] * 1000"
```

Comment:
according the task you should print array size in bytes. use sys.

Reply:
fixed !